### DISCUSSION TITLE
Git Commit Message History

### DISCUSSION THREAD

**Discussion by laktak** [extrawurst/gitui#937] [Ideas]
**Is your feature request related to a problem? Please describe.**

When my commit is aborted (e.g. by a commit hook), my commit message is lost.

**Describe the solution you'd like**

It would be nice to restore at least the last commit message. Maybe a history?

**Comment by extrawurst** [comment]
would that be a history per repo or system wide?

**Reply by laktak** [reply]
TBH while I love gitui, I started doing all my commits outside after running into too many issues (crashes/hooks).
